commander/src/commands/transaction/create/multisignature.ts

```diff
@@ -37,11 +37,10 @@ const processInputs = (
 	lifetime: number,
 	minimum: number,
 	keysgroup: ReadonlyArray<string>,
-) => ({ passphrase, secondPassphrase }: InputFromSourceOutput) =>
+) => ({ passphrase }: InputFromSourceOutput) =>
```

Comment:
this will not be release anyway with these changes, and this will be updated with pablo's PR. so the economics phase should already address this concern.